CHANGELOG.md

```diff
@@ -9,6 +9,10 @@
   - `array_flatten`
 - Added support for replicating your local Python environment on Snowflake via `Session.replicate_local_environment`.
 
+### Bug Fixes
+
+- Fixed a bug in `merge_type` logic where inferred value `nullable` was dependent in the order of input given to the function.
```

Comment:
`merge_type` is not part of public API. What about something like:

> Fixed a bug where session.create_dataframe fails to properly set nullable columns when nullability was affected the order of columns.

?

Reply:
Yep. That sounds better.